subseas_flex_fcst/config-sample.yaml

```diff
@@ -17,7 +17,7 @@ forecast_var_filePattern: CFSv2_SubXPRCP_CCAFCST_var_*.txt
 obs_filePattern: CFSv2_SubXPRCP_CCAFCST_obs_*.txt
 hcst_filePattern: CFSv2_SubXPRCP_CCAFCST_xvPr_*.txt
 variable: Precipitation Anomaly
-tp_length: 6
+target_period_length: 6
```

Comment:
I assume the target period is a week, i.e. 7 days. Then target_period_length should be 7, and the end of the period should be start + length - 1.